File: python3-memory/handler.py
import json
import datetime

def memTest(event, context):
    ts = datetime.datetime.now().timestamp()
    body = {
        "timestamp" : ts,
        "message": "Python Lambda Response. Memory Test. Function executed successfully!",
        "input": event
    }

    response = {
        "statusCode": 200,
        "body": json.dumps(body)
    }

    return response

File: node-memory/handler.js
'use strict';

module.exports.memTest = async (event, context) => {
  return {
    statusCode: 200,
    body: JSON.stringify({
      timestamp: new Date().getTime(),
      message: 'NodeJS Lambda Response. Memory Test. Function executed successfully!'
      //input: event,
    }),
  };
};
